build/Targets/ProducesNoOutput.Imports.targets

```diff
@@ -7,8 +7,9 @@
     <TargetPath></TargetPath>    <!-- Prevent projects referencing this from trying to pass us to the compiler -->
   </PropertyGroup>
   
-  <Target Name="CoreCompile" />                               <!-- Prevent Csc from being called -->  
-  <Target Name="GenerateTargetFrameworkMonikerAttribute" />   <!-- Don't generate TFM attribute -->  
+  <!-- Return this back when up-to-date bug is fixed (projects are being build everytime) -->
```

Comment:
David and Jason found a bug in build up to date check , that forced our solution to rebuild all projects every time some change happened somewhere. Roslyn repo already applied same workaround

---
In reply to: [74463316](https://github.com/dotnet/roslyn-project-system/pull/349#discussion_r74463316) [](ancestors = 74463316)